src/NuGet.Clients/NuGetConsole.Host.PowerShell/PowerShellHost.cs

```diff
@@ -74,24 +74,14 @@ internal abstract class PowerShellHost : IHost, IPathExpansion, IDisposable
         private string _currentSolutionDirectory;
 
         /// <summary>
-        /// An initial restore event used to compare against future (real) restore events. This value endures on
-        /// <see cref="_latestRestore"/> and <see cref="_currentRestore"/> as long as no restore occurs. Note that the
-        /// hash mentioned here cannot collide with a real hash.
-        /// </summary>
-        private static readonly SolutionRestoredEventArgs InitialRestore = new SolutionRestoredEventArgs(
-            isSuccess: true,
-            solutionSpecHash: "initial");
-
-        /// <summary>
         /// This field tracks information about the latest restore.
         /// </summary>
-        private SolutionRestoredEventArgs _latestRestore = InitialRestore;
+        private SolutionRestoredEventArgs _latestRestore;
```

Comment:
"Latest" and ["current"](https://github.com/NuGet/NuGet.Client/pull/1560/files#diff-43dc84d26d900804700472f5cf524260R84) are synonyms.

If I am understanding these two fields correctly, `_latestRestore` represents the previous restore and is `null` if there isn't one, whereas `_currentRestore` represents the current, in-progress restore.  Whether or not [scripts were executed for it](https://github.com/NuGet/NuGet.Client/pull/1560/files#diff-43dc84d26d900804700472f5cf524260R82) is irrelevant.

That said, `_latestRestore` should be renamed to `_previousRestore` and XML Doc Comments for both fields should be updated for clarity.